server/spec/pool_unlimited_master_spec.rb

```diff
@@ -130,11 +161,70 @@
            guest_pool = pool
         end
     end
+    
     guest_pool.should_not be nil
     @guest_client_unmapped.consume_pool(guest_pool['id'], {:uuid => @guest_unmapped.uuid, :quantity => 600})
     ents = @guest_client_unmapped.list_entitlements
     ents.size.should == 1
     ents[0].quantity.should == 600
-  end
-
+ end
+
+ it 'product multiplier should have no effect on unlimited master pool quantity' do
+   # master pool quantity expected to be -1
+   master_pool = @cp.get_pool(@pool_virt_product_multiplier['id'])
+   expect(master_pool['quantity']).to eq(-1)
+
+   # consume master pool in any quantity
+   @physical_client.consume_pool(master_pool['id'], {:quantity => 1000})
+   master_pool = @cp.get_pool(master_pool['id'])
+   expect(master_pool['quantity']).to eq(-1)
+
+   pools = @cp.list_owner_pools(@owner['key'])
+   sub_pools = pools.select do |pool|
+     pool['subscriptionId'] == @pool_virt_product_multiplier['subscriptionId'] &&
+       pool['type'] != 'NORMAL'
+   end
+   sub_pool = sub_pools[0]
+
+   # sub pool quantity is expected to be equal to virt_limit
+   sub_pool = @cp.get_pool(sub_pool['id'])
+   expect(sub_pool['quantity']).to eq(8)
```

Comment:
assertion is wrong here: master pool is unlimited (-1), so the bonus pool should also be unlimited

Reply:
Fixed.